### PR TITLE
iah integration tests

### PR DESCRIPTION
**Pull request by Kirill-K-1**
Add basic integration tests

### CODE REVIEW

File: contracts/tests/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ed25519-dalek.workspace = true
+near-sdk.workspace = true
+near-contract-standards.workspace = true
+near-primitives.workspace = true
+assert_matches.workspace = true
+workspaces.workspace = true
+near-units.workspace = true
+tokio.workspace = true
+anyhow.workspace = true
+
```
[the remark this answers is not in the document]
Comment (Kirill-K-1) [reply contracts/tests/Cargo.toml:19]: I don't think it should. This is not unit tests, but integration tests for multiple crates inside workspace. To test multiple crates inside a workspace a recommended way is to have a tests crate in workspace instead

File: contracts/tests/src/workspaces.rs [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    contract_path: &str,
+    args: near_sdk::serde_json::Value,
+) -> anyhow::Result<Contract>
+where
+    T: NetworkInfo + NetworkClient + DevNetwork + Send + Sync,
+{
+    let wasm = std::fs::read(contract_path)?;
+    let (id, sk) = worker.dev_generate().await;
+
+    let contract = worker
```
Comment (sczembor) [inline contracts/tests/src/workspaces.rs:72]: I think we can just use `dev_deploy` here and replace the `dev_generate` and `create_tla_and_deploy` with one line. We are not using the account later in here

File: contracts/tests/src/workspaces.rs [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    loop {
+        wasm = workspaces::compile_project(project_path).await?;
+        if !wasm.is_empty() || retry_count == 0 {
+            break;
+        }
+        retry_count -= 1;
+    }
+
+    let (id, sk) = worker.dev_generate().await;
+
```
Comment (sczembor) [inline contracts/tests/src/workspaces.rs:42]: same as below

File: contracts/tests/integration/test_oracle_sbt.rs [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+    let user_account = gen_user_account(&worker, "user.test.near").await?;
+    let signed_claim = build_signed_claim(
+        near_sdk::AccountId::from_str(user_account.id().as_str())?,
+        ExternalAccountId::gen(),
+        false,
+        &sec_key,
+    )?;
+
+    // TODO: add check for specific error text
```
Comment (sczembor) [inline contracts/tests/integration/test_oracle_sbt.rs:50]: I think this is done already

**Comment by sczembor** [comment]
The changes were merged in this PR https://github.com/near-ndc/i-am-human/pull/90. Since it was left for so long it was easier to create a new one

**Pull request closed without merging**